### PR TITLE
Add cookie banner

### PR DESCRIPTION
**Pull request by sweta1308**
Add cookie banner for google tag manager.
image

### CODE REVIEW

File: src/components/CookieModal.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        setCookieValue(choice);
+    }, []); 
+    
+    useEffect(() => {
+        const userCookie = localStorage.getItem('userCookie');
+        setCookieValue(userCookie);
+        setLoading(false);
+    }, []);
+
+    if (loading) return null;
```
Comment (chirag-madlani) [inline src/components/CookieModal.tsx:18]: We can combine condition here for loading and coockieValue no?

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -11,13 +11,21 @@ import '@fontsource/metropolis/700.css';
...
 
 config.autoAddCss = false;
 
 export default function App({ Component, pageProps }: AppProps) {
+  const [storedCookie, setStoredCookie] = useState<string | null>(null);
+
+    useEffect(() => {
+        const userCookie = localStorage.getItem('userCookie')
+        setStoredCookie(userCookie);
+    }, []);
```
Comment (chirag-madlani) [inline src/pages/_app.tsx:27]: What happen when user Do a reject, will this re-eval at that time?

Comment (sweta1308) [reply src/pages/_app.tsx:27]: No. It won't. But it should to get the latest value.

File: src/components/CookieModal.tsx (hunk 1 above)
Comment (sweta1308) [reply src/components/CookieModal.tsx]: Yeah we can. Updated that,

File: src/pages/_app.tsx (hunk 2 above)
Comment (sweta1308) [reply src/pages/_app.tsx:27]: Updated

**Review by chirag-madlani — COMMENTED** [review]
@jpnguyen Can you help us with Privacy Policy page?

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**